scripts/data_utils.py

```diff
@@ -489,10 +706,16 @@ def chunk_content_helper(
                 splitter = PythonCodeTextSplitter.from_tiktoken_encoder(
                     chunk_size=num_tokens, chunk_overlap=token_overlap)
             else:
-                splitter = RecursiveCharacterTextSplitter.from_tiktoken_encoder(
-                    separators=SENTENCE_ENDINGS + WORDS_BREAKS,
-                    chunk_size=num_tokens, chunk_overlap=token_overlap)
-            chunked_content_list = splitter.split_text(doc.content)
+                if file_format == "html":

```

Comment:
we should create a different format for pdfs and handle them separately than html - are we sure how html tables look like? Does it affect how htmls are processed too?